tests/integration/security/authorization_test.go

```diff
@@ -128,97 +131,88 @@ func TestAuthorization_JWT(t *testing.T) {
-						newTestCase(d, "[Token1WithAzp]", jwt.TokenIssuer1WithAzp, "/valid-jwt", true),
-						newTestCase(d, "[Token1WithAud]", jwt.TokenIssuer1WithAud, "/valid-jwt", true),
-
-						// Test condition "request.auth.presenter" on suffix "/presenter".
-						newTestCase(d, "[Token1]", jwt.TokenIssuer1, "/request/presenter", false),
-						newTestCase(d, "[Token1WithAud]", jwt.TokenIssuer1, "/request/presenter", false),
-						newTestCase(d, "[Token1WithAzp]", jwt.TokenIssuer1WithAzp, "/request/presenter-x", false),
-						newTestCase(d, "[Token1WithAzp]", jwt.TokenIssuer1WithAzp, "/request/presenter", true),
-
-						// Test condition "request.auth.audiences" on suffix "/audiences".
-						newTestCase(d, "[Token1]", jwt.TokenIssuer1, "/request/audiences", false),
-						newTestCase(d, "[Token1WithAzp]", jwt.TokenIssuer1WithAzp, "/request/audiences", false),
-						newTestCase(d, "[Token1WithAud]", jwt.TokenIssuer1WithAud, "/request/audiences-x", false),
-						newTestCase(d, "[Token1WithAud]", jwt.TokenIssuer1WithAud, "/request/audiences", true),
-					}
-					rbacUtil.RunRBACTest(t, cases)
-				})
+						cases := []rbacUtil.TestCase{
+							newTestCase(dst, "[NoJWT]", "", "/token1", false),
+							newTestCase(dst, "[NoJWT]", "", "/token2", false),
+							newTestCase(dst, "[Token1]", jwt.TokenIssuer1, "/token1", true),
+							newTestCase(dst, "[Token1]", jwt.TokenIssuer1, "/token2", false),
+							newTestCase(dst, "[Token2]", jwt.TokenIssuer2, "/token1", false),
+							newTestCase(dst, "[Token2]", jwt.TokenIssuer2, "/token2", true),
+							newTestCase(dst, "[Token1]", jwt.TokenIssuer1, "/tokenAny", true),
+							newTestCase(dst, "[Token2]", jwt.TokenIssuer2, "/tokenAny", true),
+							newTestCase(dst, "[PermissionToken1]", jwt.TokenIssuer1, "/permission", false),
+							newTestCase(dst, "[PermissionToken2]", jwt.TokenIssuer2, "/permission", false),
+							newTestCase(dst, "[PermissionTokenWithSpaceDelimitedScope]", jwt.TokenIssuer2WithSpaceDelimitedScope, "/permission", true),
+							newTestCase(dst, "[NestedToken1]", jwt.TokenIssuer1WithNestedClaims1, "/nested-key1", true),
+							newTestCase(dst, "[NestedToken2]", jwt.TokenIssuer1WithNestedClaims2, "/nested-key1", false),
+							newTestCase(dst, "[NestedToken1]", jwt.TokenIssuer1WithNestedClaims1, "/nested-key2", false),
+							newTestCase(dst, "[NestedToken2]", jwt.TokenIssuer1WithNestedClaims2, "/nested-key2", true),
+							newTestCase(dst, "[NestedToken1]", jwt.TokenIssuer1WithNestedClaims1, "/nested-2-key1", true),
+							newTestCase(dst, "[NestedToken2]", jwt.TokenIssuer1WithNestedClaims2, "/nested-2-key1", false),
+							newTestCase(dst, "[NestedToken1]", jwt.TokenIssuer1WithNestedClaims1, "/nested-non-exist", false),
+							newTestCase(dst, "[NestedToken2]", jwt.TokenIssuer1WithNestedClaims2, "/nested-non-exist", false),
+							newTestCase(dst, "[NoJWT]", "", "/tokenAny", false),
+							newTestCase(c, "[NoJWT]", "", "/somePath", true),
```

Comment:
this still sends to `c`?

Reply:
Yes, this is existed in the original test. https://github.com/istio/istio/blob/ae7600577b1ea13719904e53c0eac422ca7df214/tests/integration/security/authorization_test.go#L200

There is no security policy applied to `c` https://github.com/istio/istio/blob/master/tests/integration/security/testdata/authz/v1beta1-jwt.yaml.tmpl

I think we send request to `c` as a comparison with the targets that have security policies deployed.
There are two duplicated test cases of `newTestCase(c, "[NoJWT]", "", "/somePath", true),`

I found that Tairan previously duplicated test cases by mistake, and I confirmed with Tairan that I can removed the duplicated ones.